runtime/lua/vim/lsp/semantic_tokens.lua

```diff
@@ -576,6 +585,76 @@ function M.stop(bufnr, client_id)
   end
 end
 
+--- Return the semantic token(s) at the given position.
+--- If called without argument, returns the token under the cursor.
+---
+---@param bufnr number|nil Buffer number (0 for current buffer, default)
+---@param row number|nil Position row (default cursor position)
+---@param col number|nil Position column (default cursor position)
+---
+---@return table[]|nil tokens Table of tokens at position
```

Comment:
```suggestion
---@return table[] tokens Table of tokens at position
```

Could treat `nil` the same as "empty tokens" to reduce the need for handling an extra case on call-sites.

Reply:
Hmm, downstream kinda relies on that special handling. I also wanted to make the distinction between "no tokens available" and "no tokens at this position".